src/components/ExportCSV/ExportCSV.jsx

```diff
@@ -0,0 +1,326 @@
+import React from 'react';
+import { ChakraProvider, Select, Button, Input, FormLabel } from '@chakra-ui/react';
+
+import './ExportCSV.css';
+
+function ExportCSV() {
+  const [sort, setSort] = React.useState('ascend');
+  const [boxes, setBoxes] = React.useState('all');
+  const [date, setDate] = React.useState('all');
+  const [launch, setLaunch] = React.useState('both');
+  const [zip, setZip] = React.useState('all');
+  const [boxesCustom, setBoxesCustom] = React.useState('');
+  const [dateCustom, setDateCustom] = React.useState('');
+  const [startCustom, setStartCustom] = React.useState('');
+  const [endCustom, setEndCustom] = React.useState('');
+  const [zipCustom, setZipCustom] = React.useState('');
+
+  // Function to check valid range
+  function checkRange(value) {
```

Comment:
Also for all the functions in this file - could you please add brackets for your if/else statements?
e.g.:
if (condition) {
  // statements
} else {
  // else statements
}